source/testnet/references/manage-accounts.rst

```diff
@@ -43,43 +38,57 @@ byte 1. An example of such an address is
+The name is resolved to an address before interactions with the node.
 
-The mobile wallet: Concordium ID
-================================
+Account concepts
+================
 
-`Concordium ID`_ is a smartphone app that allows creation of accounts from
-identities issued by the external identity provider `Notabene`_, as well as
-transfers of GTU from created accounts.
+.. _managing-account-balances:
 
-This app can export identities and accounts that the user created.
+Account balances
+----------------
 
-Command-line tool
-=================
+Each account has two balances, the *public balance* which can be *seen* by every
+member of the Concordium Network, and a :ref:`shielded
+balance<glossary-shielded-balance>`, which is also referred to as *encrypted
+balance*. The shielded balance is only known to the owner of the account. It
+can only be used in :ref:`shielded transfers<glossary-shielded-transfer>` to
+other accounts.
 
-The Concordium distribution ships with a command-line tool named
-concordium-client_. It is designed as a low-level interface to the
-Concordium blockchain. It cannot be used to create identities, but it can
-`import`_ accounts exported from the mobile wallets. Once an account has been
-imported, the tool can be used to do GTU transfers from the account, as well as
-send all other `transaction`_ types supported by the Concordium blockchain.
+The public balance of the account is used for payment of transaction fees,
+baking, finalization, and transfers.
+
+At any given time some of the public balance might be unavailable for use. This
+can happen in two ways.
+
+- the account has :ref:`staked<glossary-staked-amount>` some (or all) of the public
+  balance in order to become a baker
+- some of the public balance is locked-up because it was received via a
+  :ref:`transfer with schedule<glossary-transfer-with-schedule>`
 
-Node Dashboard
-==============
+The locked-up balance can only be staked, but it can not be used for payment of
+transaction fees, nor can it be transferred to other accounts.
 
-The node dashboard can import accounts from the files exported from the wallet.
-The dashboard only uses the account to register the node as a baker.
+The relationship between the different balances can be best illustrated on an
+example.\ [#footnote-ignore-fees]_ Suppose that on January 1 the account starts
+with 100 GTU on the public balance. None of it is locked-up or staked.
+
+Then on January 2 the account receives 50 GTU via a :ref:`transfer with
+schedule<glossary-transfer-with-schedule>` with the release scheduled for
+December 31 of the same year. At this point, January 2, the account has 50 GTU
+at disposal, the rest being locked. If the account tried to transfer more than
+50 GTU the transaction would be rejected.
+
+On January 3 the account becomes a baker with the initial stake of 125 GTU.
+This is successful because the total public balance is 150GTU.
+After this the account still has 25 GTU at disposal.
```

Comment:
Perhaps I'm reading this wrong @abizjak but I got a bit lost here – it says it starts with 100 then gets a 50GTU transfer, but then "At this point, January 2, the account has 50 GTU
at disposal, the rest being locked. "

Shouldn't this read "At this point, January 2, the account has 100 GTU
at disposal, the remaining 50 being locked" ?

Reply:
Yes, you are right, silly typos.